download_and_install/ola_on_raspberry_pi.md

```diff
@@ -0,0 +1,68 @@
+OLA on Raspberry Pi
+===================
+
+This tutorial describes how to get OLA running on the
+[Raspberry Pi](https://www.raspberrypi.org/). The procedure described here is designed to get OLA up and running as fast
+as possible. There is plenty of information on the [Raspberry Pi Docs](https://www.raspberrypi.org/documentation/).
+The [Raspberry Pi Forum](https://www.raspberrypi.org/forums/) is a good place to ask for help on Raspberry Pi specific
+issues.  
+
+Getting Started
+===============
+
+See the official [setup guide](https://www.raspberrypi.org/documentation/setup/) for hardware requirements.
+
+- A monitor is not strictly required to use OLA, but one can be helpful for debugging purposes.
+- If you plan on using a USB DMX interface that does not use its own power supply, you will also need a powered USB hub.
+  The Raspberry Pi hardware cannot supply enough power over USB to meet the needs of most USB DMX interfaces.
+- A wired network connection is required when using OLA with any network protocols. DMX over WiFi is not Wireless DMX.
+
+Installation
+=============
+
+OLA is easily available when using [Raspberry Pi OS](https://www.raspberrypi.org/software/operating-systems/). If your
+Raspberry Pi will only be used for OLA, use the "Raspberry Pi OS Lite" operating system.
+
+Follow the official installation instructions
+[here](https://www.raspberrypi.org/documentation/installation/installing-images/) to install the OS.
+
+Starting Up
+===========
+
+If you don't have a display and keyboard, follow the SSH instructions
+[here](https://www.raspberrypi.org/documentation/remote-access/ssh/README.md). This guide will assume you are typing
+directly into the Raspberry Pi or have established an SSH session.
+
+Installing
+----------
+
+As with other Debian Linux systems, installing OLA can be performed by running
+
+    sudo apt-get update
+    sudo apt-get install ola
+
+in the terminal.
+
+Reboot the device with `sudo reboot` to begin using OLA.
```

Comment:
Why would this be required? Doesn't simply starting the service suffice?

Reply:
Since olad runs on startup, this seemed more clear than explaining why the user only needs to start the service manually this one time.  It creates the impression that the Pi, at this point, can be treated more as an appliance that need only be turned on and off.

Reply:
Indeed, that makes sense